exasol/ai/mcp/server/mcp_server.py

```diff
@@ -109,11 +116,13 @@ def _register_tools(self):
                 self.describe_table,
                 description=(
                     "Describes the specified table in the specified schema of the "
-                    "Exasol Database. Returns the list of table columns. For each "
-                    "column provides the name, the data type, an optional comment, "
-                    "and the primary key flag. If the column is a foreign key the "
-                    "information about the referenced column is provided. It includes "
-                    "the schema, the table and the column name in parentheses."
+                    "Exasol Database. The description includes the list of columns "
+                    "and the list of constraints. For each column provides the name, "
+                    "the data type and an optional comment. For each constraint "
+                    "provides its type, e.g. PRIMARY KEY, the list of columns the "
+                    "constraint is applied to an an optional name. For a FOREIGN KEY "
+                    "also provides the referenced schema, table and a list of columns "
+                    "in the referenced table."
```

Comment:
Stupid question: Can keys have comments

Reply:
I don't think so. At least there is no such column in the `SYS.EXA_ALL_CONSTRAINT_COLUMNS`